app/styles/_header.scss

```diff
@@ -0,0 +1,39 @@
+header {
+  width: 100%;
+  height: 20%;
+  background: url('http://www.lorempixel.com/1600/900/cats/');
+  background-repeat: no-repeat;
+  text-align: center;
+  filter: grayscale(80%);
+
+
+  nav {
+    ul {
+      float: right;
+      list-style: none;
+    }
+
+    li {
+      display: inline;
+      margin: 1rem;
+    }
+  }
+
+  h1 {
+    clear: both;
+  }
+}
+
+
+
+
+.nav-container {
+  width: 70%;
```

Comment:
The max width of this should match your container so that the top nav lines up with the other content edges of the page.